### PR TITLE
fix(hot): preload HMR plugin into compiler options.

### PR DESCRIPTION
**Pull request by joaovieira**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| Breaking change?  | no
| New feature?      | no
| Deprecations?     | no
| Tests added?      | no
| Docs updated?     | no
| Fixed tickets     | #121
| License           | MIT

As per the docs ([here](https://webpack.github.io/docs/webpack-dev-server.html#hot-module-replacement) and [here](https://github.com/webpack/webpack-dev-server/issues/97#issuecomment-70388180)), the plugin should be added to the webpack configuration options.

I don't know if it would be possible by applying the plugin directly as you do, but this definitely fixes the issue :)

With the current version `module.hot` it not defined in the bundle, which indicates the plugin wasn't loaded. With this change it is and everything works as expected.

### CODE REVIEW

**Comment by danez** [comment]
Thanks.

File: tasks/webpack-dev-server.js [hunk 1]
```diff
@@ -45,7 +43,10 @@ yarn add webpack-dev-server`);
       const devClient = [
         `webpack-dev-server/client?${protocol}://${opts.host}:${opts.port}`
       ];
-      if (opts.hot) devClient.push('webpack/hot/dev-server');
+      if (opts.hot) {
+        devClient.push('webpack/hot/dev-server');
+        webpackOptions.plugins.plush(new webpack.HotModuleReplacementPlugin());
```
Comment (octalmage) [inline tasks/webpack-dev-server.js]: Should this be plush or push? Just noticed this, not sure if plush is a thing. 
.

Comment (joaovieira) [inline tasks/webpack-dev-server.js]: Yes. This was fixed in #127 already. And plush is a thing:
standard-1024x1024 2

Comment (octalmage) [inline tasks/webpack-dev-server.js]: haha awesome.

**Pull request merged**